app/code/BVMyBlog/Blog/Block/LastPosts.php

```diff
@@ -0,0 +1,116 @@
+<?php
+declare(strict_types=1);
+
+namespace BVMyBlog\Blog\Block;
+
+use BVMyBlog\Blog\Model\BlockRepository;
+use Magento\Catalog\Model\Locator\RegistryLocator;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\Api\SortOrderBuilder;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Exception\NotFoundException;
+use Magento\Framework\UrlInterface;
+use Magento\Framework\View\Element\Block\ArgumentInterface;
+use Magento\Framework\View\Element\Template;
+
+/**
+ * Returns 5 latest posts
+ */
+class LastPosts extends Template implements ArgumentInterface
```

Comment:
Template is **Block**. ArgumentInterface is used to define **View Model**. Please use block or view model, but not both